src/test/java/org/breedinginsight/services/BrAPIGermplasmServiceUnitTest.java

```diff
@@ -52,6 +52,8 @@ void setup() {
         programService = mock(ProgramService.class);
     }
 
+    //TODO: Refactoring in BI-1443 needs to be done before this can be enabled due to BrAPIDAOUtil.search currently being static
+    /*
```

Comment:
What changed that this test can no longer run?

Reply:
That was related to a bugfix from BI-1088 because of the introduction of ProgramCache altering getting germplasm by name for export. Seems I must have pulled from release somehow since I was still waiting on reviews to get that fix merged to develop. Should be merged now